src/components/Widgets/Widget.style.ts

```diff
@@ -19,9 +21,14 @@ export const WidgetWrapper = styled(Box, {
     },
   }),
   zIndex: 2,
-  // [theme.breakpoints.up('sm' as Breakpoint)]: {
-  //   maxWidth: 416,
-  // },
+
+  [theme.breakpoints.up('sm' as Breakpoint)]: {
+    ...(!isExpanded && {
+      '& > div': {
+        height: 616,
```

Comment:
This is the minimal widget height we should enable - without Wallet-Address field we can go as low as 496px:
![image](https://github.com/user-attachments/assets/e88a2d2c-9d40-42c6-820c-a9a57f45b631)